pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -21,15 +21,14 @@ import (
 	"math"
 
 	"k8s.io/client-go/util/workqueue"
+	kubeschedulerconfig "k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
 
-const parallelism = 16
```

Comment:
We still need a default of 16. See `pkg/scheduler/apis/config/v1beta1/defaults.go` to do that.